acceptance/setup/packages/pre-suite/010_Install.rb

```diff
@@ -48,5 +48,15 @@
 install_packages_on(master, MASTER_PACKAGES)
 install_packages_on(agents, AGENT_PACKAGES)
 
+agents.each do |agent|
+  if agent['platform'] =~ /windows/
+    arch = agent[:ruby_arch] || 'x86'
+    base_url = "http://builds.puppetlabs.lan/puppet/#{ENV['SHA']}/artifacts/windows"
+    filename = "puppet-agent-#{ENV['VERSION']}-#{arch}.msi"
```

Comment:
We only recently added `-#{arch}` to x86.

So you can see builds from 2/5 have it:
http://builds.puppetlabs.lan/puppet/09b0ece5f6f8a5c836f0fa518ca7455126825293/artifacts/windows/

But not from 1/29:
http://builds.puppetlabs.lan/puppet/d75d56332b53e7a252cd1550736c2b9964ec9b14/artifacts/windows/

Might want to add some safeguards here for when `-x86` wasn't part of the package name, for the sake of backward compat.